modules/cli/src/main/scala/scala/cli/commands/config/Config.scala

```diff
@@ -285,6 +278,15 @@ object Config extends ScalaCommand[ConfigOptions] {
                       else
                         values
 
+                    val previousValue = db.get(entry).wrapConfigException.orExit(logger)
+
+                    confirmUpdatedValue(
```

Comment:
I think this makes almost all config writes interactive now (except for `Keys.repositoryCredentials` and `Keys.publishCredentials`). We could limit that to only Java props key, or only to `StringListEntry` keys.

IMO for keys that have just a single value, so our `BooleanEntry`, `PasswordEntry` or `StringEntry`, asking for confirmation is not that useful.
WDYT?

Reply:
Yes, that sounds like a good idea. so, I limit it only to `StringListEntry`.